src/app/web-analytics.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+
+declare let gtag: Function;
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WebAnalyticsService {
+
+  constructor() { }
+
+  public eventEmitter(
```

Comment:
I'd prefer this to be emitAnalyticsEvent so that it's a verb to make usage clearer.